src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -940,10 +949,17 @@ private void LoadAndCreateConnectors(OrderedDictionary inportConnections, Ordere
                 {
                     if (outportConnections[varName] != null)
                     {
-                        foreach (var endPortModel in (outportConnections[varName] as List<PortModel>))
+                        foreach (var oldConnector in (outportConnections[varName] as List<ConnectorModel>))
                         {
+                            var endPortModel = oldConnector.End;
                             NodeModel endNode = endPortModel.Owner;
                             var connector = ConnectorModel.Make(this, endNode, i, endPortModel.Index);
+                            //during an undo operation we should set the new input connector
```

Comment:
Comments to be updated here, should be `output connector`?

Reply:
thanks - will fix.